client/signup/config/flows-pure.js

```diff
@@ -153,10 +153,10 @@ export function generateFlows( {
 		},
 		{
 			name: 'onboarding-with-email',
-			steps: getAddOnsStep( [ 'user', 'domains', 'emails', 'plans' ] ),
+			steps: getAddOnsStep( [ 'user', 'domains', 'mailbox', 'plans' ] ),
 			destination: getSignupDestination,
 			description:
-				'Copy of the onboarding flow that always includes an email step; the flow is used by the Professional Email landing page',
+				'Copy of the onboarding flow that includes non-skippable domain and email steps; the flow is used by the Professional Email landing page',
 			lastModified: '2021-08-11',
```

Comment:
Maybe we should update this information?
```suggestion
			lastModified: '2022-09-06',
```

Reply:
Good point! It's actually updated in #66921 though